test/runtime/observability/cluster_log_forwarder.go

```diff
@@ -127,7 +127,7 @@ func (p *PipelineBuilder) Named(name string) *PipelineBuilder {
 	return p
 }
 
-func (p *PipelineBuilder) ToElasticSearchOutput(visitors ...func(output *obs.OutputSpec)) *ClusterLogForwarderBuilder {
+func (p *PipelineBuilder) ToElasticSearchOutput(auth obs.HTTPAuthentication, visitors ...func(output *obs.OutputSpec)) *ClusterLogForwarderBuilder {
```

Comment:
If we require auth here should we also mark the API as required?  I chose this pattern for one of my recent submitssions because we "required" auth.  I'm open to leaving this here instead of deferring a user to use a visitor

Reply:
We can keep with the visitor instead.